administration.html

```diff
@@ -0,0 +1,272 @@
+                        "Maitenant que le traitement est automatisé, j'ai enfin le temps d'aider les usagers en
+                        difficulté, dans certains cas je peux les contacter de manière proactive."
+                    </p>
+                    <p style="text-align: right;">
+                        <i>- Aurélie, directrices affaires générales, Clamart</i>
+                    </p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card__content">
+                    <p style="font-size: 1.3em; text-align: center">
+                        <b>563 178</b> justificatifs papier évités pour le mois de septembre 2018
+                    </p>
+                </div>
+            </div>
+        </div>
+
+        <p class="section__subtitle"></p>
+    </div>
+</section>
+
+<section class="section-white">
+    <div class="container" style="font-size: x-large">
+        <p>
+            La lenteur d'une démarche administrative vient du traitement des pièces justificatives. Un usager doit régulièrement se rendre dans sa mairie pour fournir des informations qu'il a déjà transmis à d'autres administrations comme : son revenu fiscal de référence, son quotient familial CAF, la composition de sa famille, etc.
+            Les pièces justificatives rendent les démarches plus complexes, augmentent les risques d'erreur et demandent un traitement manuel de chaque dossier.
+        </p>
+        <p>
+            Faire un formulaire en ligne pour soumettre ses pièces justificatives ne résout pas le problème : un agent
+            doit toujours passer du temps pour valider et saisir l'information fournie par l'usager. API Particulier va
+            plus loin et permet de récupérer directement l'information à la source dans les bases de la DGFIP, de la
+            CAF, etc. L'information est certifiée et vous affranchit du besoin de demander et de valider des pièces
+            justificatives. Votre démarche est alors 100% dématérialisée.
+        </p>
+        <p>
+            Vous pouvez utiliser API Particulier pour toutes vos démarches : transport, cantine, culture, sport, etc. À partir des informations fiscale, familiale et géographique de vos
+            usagers, vous pouvez calculer automatiquement les montants de vos aides.
+        </p>
+        <p>
+            Nous voudrions banir l'emploi de pièces justificatives dans votre administration. Cela dit, si vous avez la moindre question avant de
+            commencer,
+            <a href="mailto:contact@particulier.api.gouv.fr?subject=Contact%20via%20particulier.api.gouv.fr%2Fadministration">n'hésitez
+                pas à nous contacter</a>.
+        </p>
+    </div>
+</section>
+
+<section class="">
+    <div class="container">
+        <p class="section__subtitle">
+            <button class="button" onclick="document.getElementById('gform-input-name').focus()">
```

Comment:
The `focus` makes the form appear on the top of the page which takes away some of the form's context.
<img width="926" alt="capture d ecran 2018-09-24 a 14 43 25" src="https://user-images.githubusercontent.com/13916213/45952763-4d108600-c008-11e8-993c-97c0a720d48e.png">

It might be clearer if the button transport the user to the top of the page

![image](https://user-images.githubusercontent.com/13916213/45952843-91038b00-c008-11e8-8678-76f447798617.png)

Reply:
Well spotted ! It's indeed the case in firefox, but not in chrome. I was looking for the simplier solution to accomplish that and it's not perfect.